### PR TITLE
add DEFAULT_WAIT_FOR_IFRAME_TIMEOUT and custom timeout for waitForIframe function. Bump version

### PR DESCRIPTION
**Pull request by miherlosev**
@AlexanderMoskovkin 
After merge one need to update readme.md

### CODE REVIEW

File: src/templates/globals.mustache.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 var Promise = require('pinkie');
 
-var WAIT_TIMEOUT             = 3000;
-var WAIT_FOR_IFRAME_TIMEOUT  = 10000;
-var CHECK_PREDICATE_INTERVAL = 50;
+var WAIT_TIMEOUT                    = 3000;
+var DEFAULT_WAIT_FOR_IFRAME_TIMEOUT = 10000;
```
Comment (AlexanderMoskovkin) [inline src/templates/globals.mustache.js]: WAIT_FOR_IFRAME_TIMEOUT_DEFAULT

**Review by AlexanderMoskovkin — CHANGES_REQUESTED** [review]

File: src/templates/globals.mustache.js (hunk 1 above)
Comment (miherlosev) [reply src/templates/globals.mustache.js]: After discussing we decide to rollback to WAIT_FOR_IFRAME_TIMEOUT

File: src/templates/globals.mustache.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ window.QUnitGlobals = {
                 iframe.removeEventListener('load', loadEventHandler);
                 ok(false, 'Timeout error');
                 start();
-            }, WAIT_FOR_IFRAME_TIMEOUT);
+            }, timeout || WAIT_FOR_IFRAME_TIMEOUT);
```
Comment (AlexanderMoskovkin) [inline src/templates/globals.mustache.js]: it doesn't work if `timeout === 0`

so, it should be 

``` js
timeout === void 0 ? WAIT_FOR_IFRAME_TIMEOUT : timeout
```

**Review by AlexanderMoskovkin — CHANGES_REQUESTED** [review]

File: src/templates/globals.mustache.js (hunk 2 above)
Comment (miherlosev) [reply src/templates/globals.mustache.js]: Fixed

**Review by AlexanderMoskovkin — APPROVED** [review]

**Comment by AlexanderMoskovkin** [comment]
I'll publish this in few minutes

**Comment by miherlosev** [comment]
Also need to update README.md (add description about new settings)

**Pull request merged**